addons/sale/static/tests/sales_team_dashboard_tests.js

```diff
@@ -59,10 +59,10 @@ QUnit.test("edit target with several o_kanban_primary_bottom divs", async (asser
             <kanban>
                 <field name="invoiced_target"/>
                 <templates>
-                    <div t-name="kanban-box" class="container o_kanban_card_content">
+                    <div t-name="kanban-card" class="container o_kanban_card_content">
```

Comment:
The test seems related to some legacy behavior. Maybe it has become useless and should be removed. Or should it be adapted? What do you think @aab-odoo?

Reply:
Indead, I would
- rename the test into "edit progressbar target"
- remove the two empty (unnecessary) divs .o_kanban_primary_bottom from the arch